utilities-examples/cloudify_rest/example-1-blueprint.yaml

```diff
@@ -0,0 +1,198 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/5.0.5/types.yaml
+  - plugin:cloudify-utilities-plugin
+
+inputs:
+
+  rest_endpoint:
+    description: >
+      REST API endpoint
+    default: "jsonplaceholder.typicode.com"
+
+  rest_endpoint_cert_path:
+    description: >
+      REST API endpoint CA_BUNDLE
+    # ubuntu
+    # default: /usr/lib/python2.7/dist-packages/certifi/cacert.pem
+    # centos
+    # default: /opt/manager/env/lib/python2.7/site-packages/certifi/cacert.pem
+    # or get cacert path/content from secret
+    default: { get_secret: cacert_bundle }
+
+  rest_endpoint_cert:
+    description: >
+      REST API endpoint CA_BUNDLE
```

Comment:
Let's get rid of the cert content here. We can either put it in an inputs file, or generate it w/ a script, or use a secret.